File: week2/SpecialLotto.java


import java.util.Random;
import java.util.ArrayList;
import java.util.Collections;

public class SpecialLotto {

    public static ArrayList<Integer> genMagickNumList() {
        ArrayList<Integer> mynum = new ArrayList<>(71);

        for (int i = 1; i < 71; i++) {
            mynum.add(i);
        }
        return mynum;
    }

    public static ArrayList<Integer> getTicket() {
        ArrayList<Integer> lotto = new ArrayList<>();
        Random rn = new Random();
        ArrayList<Integer> a = genMagickNumList();
        int z = 0;
        while (z < 5) {
            int index = rn.nextInt(a.size() - 1);
            int value = a.get(index);
            lotto.add(value);
            a.remove(index);
            z++;
        }
        Collections.sort(lotto);
        lotto.add(rn.nextInt(25)+1);

        return lotto;
    }

    public static void main(String [] args) {
        int size=10;
        ArrayList<ArrayList<Integer>> a = new ArrayList<ArrayList<Integer>>(size);
        for (int i=0; i < size; i++) {
            a.add(getTicket());
        }
        for (ArrayList<Integer> i : a ) {
            System.out.println(i);
        }
    }
}

File: week2/SpecialLotto.php
<?php

function genMagickNumList() {
	$mynum = array();

    for($i = 1; $i < 71; $i++) {
		array_push($mynum, $i);

    }
    return $mynum;
}

function getTicket() {
    $lotto = array();
        
    $a = genMagickNumList();
    $z = 0;
	
    while ($z < 5) {
        $index = rand(0, sizeof($a)-1);
        $value = $a[$index];
		array_push($lotto, $value);

        $z++;
    }
	

    array_push($lotto, rand(1,26));
	
    return $lotto;
}
	
$size=10;
$a = array();

for ($i=0; $i < $size; $i++) {
    array_push($a, getTicket());
    }
	
foreach($a as $m) {
    echo "Lotto Number:\t";
	foreach ($m as $num){
		echo $num . "\t";
    }
	echo "\n\n";
}

?>